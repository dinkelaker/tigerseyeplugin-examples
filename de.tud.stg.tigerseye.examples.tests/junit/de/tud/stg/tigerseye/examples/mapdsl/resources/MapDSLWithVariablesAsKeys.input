int a = 1
int b = 2
Map m = [Integer, String: a="Hans", b="Peter"]	