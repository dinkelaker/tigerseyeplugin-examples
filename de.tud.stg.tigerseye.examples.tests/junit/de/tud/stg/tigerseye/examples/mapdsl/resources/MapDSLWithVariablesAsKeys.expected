int a = 1
int b = 2
Map m = buildMap(Integer.class,String.class, [
             buildEntry(a,"Hans") as Entry[] ,
              buildEntry(b,"Peter") as Entry[] ])